### PR TITLE
Added accordion arrows + various style changes to text.

### PR DESCRIPTION
**Pull request by stonespheres**
I've added the accordion arrows and changed the accordion button states. We can change the arrows used, for now they're set using Font Awesome icons, as such I've had to import the style sheet.
Check line 46 as an example for the button states, I've removed the ``collapsed`` parameter so the CSS styling will work as intended.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -126,7 +126,14 @@ button.btn-link.btn-block.text-left::after {
     top: 6px;
 }
 
-button.btn-link.btn-block.text-left.collapsed::after {
+button.btn-link.btn-block.text-left::after {
```
Comment (stonespheres) [inline style.css:132]: Changed the orientation of the arrows. I had difficulty getting the arrows to sit uniformly down when the page loaded. To fix this I altered some code in index.html where `button.btn-link.btn-block.text-left.collapsed` was defined. The problem itself was that one button element was already set to `collapsed`. All elements now only enter the `collapsed` state after the accordion has been checked.

File: style.css [hunk 2]
```diff
@@ -126,7 +126,14 @@ button.btn-link.btn-block.text-left::after {
...
+    font-family: "Font Awesome 5 Free";
+    content: "\f078";
+    font-weight: bold;
+    font-size: 22px;
+    position: absolute;
+    right: 10px;
+    top: 6px;
```
Comment (stonespheres) [inline style.css]: Icons wouldn't load without correct styling. Position had to be defined in order to keep the page clean and uniform.

**Comment by ZsuzsaMano** [comment]
Thank you, I will review it in the next days, and merge it.

**Review by stonespheres — COMMENTED** [review]
> Thank you, I will review it in the next days, and merge it.

No worries, I'll be keeping tabs for any other issues I can help out with. Let me know any changes you'd like moving forward.

File: style.css [hunk 3]
```diff
@@ -38,7 +40,8 @@ main {
     min-height: 90vh;
 }
 
-h1 {
+h1,
+h2 {
```
Comment (stonespheres) [inline style.css:44]: Added h2 elements to add styling to them. I consciously put these together to use less code, however the padding defined here `padding: 1em;` causes the accordion element to be a bit larger. I suggest we have a look at separating these elements.

File: style.css [hunk 4]
```diff
@@ -27,8 +27,10 @@ a {
 }
 
 .navbar-dark .navbar-nav .nav-link:focus,
-.navbar-dark .navbar-nav .nav-link:hover {
+.navbar-dark .navbar-nav .nav-link:hover,
+.nav-link {
     color: var(--font-color);
+    font-family: Get Schwifty;
```
Comment (stonespheres) [inline style.css]: `nav-link` added to the style sheet. Links in the `navbar` now render with the 'Get Schwifty' font. This looks pretty good, but creates a bit of discrepancy between the navigation bar and the page content.

**Review by stonespheres — COMMENTED** [review]
Minor changes to styling in this commit, mainly navigation bar links now format with the ``Get` Schwifty`` font family as well as ``h2`` elements.

**Pull request merged**